modules/ROOT/pages/to-find-info.adoc

```diff
@@ -4,336 +4,120 @@ include::_attributes.adoc[]
-
-
-The original Exchange search experience uses query language to search assets by tags, categories, and custom fields.
-
-=== Search by Tags
-
-Search by tags using either `tag:"some value"` or `tag:value`.
-
-If the value has no spaces, then double quotes are optional.
-
-Tag search is case-insensitive.
-
-Examples:
-
-[%header%autowidth.spread]
-|===
-|Search |Tags |Matches?
-|tag:"some value" |some value |Yes
-|tag:"some value" |value |No
-|tag:"value" |value |Yes
-|tag:value |value |Yes
-|tag:VALUE |value |Yes
-|tag:VaLuE |value |Yes
-|tag:value |val |No
-|===
-
-=== Search by Categories
-
-Search by categories using `category:"some key" = "some value"`.
-
-If the category has no spaces in the key or value, then double quotes are optional.
-
-The key can include the asterisk (`*`) regular expression to search for characters within a category name.
-
-The key is case-sensitive.
-
-A search value matches an asset value if either of these is true:
+Categories are specific groupings of assets set by the administrator of a business group or an organization owner.
+* Tag
++
+Keyword or term that describes a specific search. 
+* Lifecycle state
++
+Lifecycle state can be stable, deprecated, or development.
 
-* The search value is entirely lowercase.
-* The letter case of the search value and the asset value match.
+Filter options appear when they are available within an organization. 
 
-Examples:
+To filter search results:
 
-[%header%autowidth.spread]
-|===
-|Search |Category |Matches?
-|category:my-key = my-value |my-key:my-value |Yes
-|category:my-key = MY-VALUE |my-key:MY-VALUE |Yes
-|category:my-key = my-value |my-key:MY-VALUE |Yes
-|category:"my key" = "my value" |my key:my value |Yes
-|category:"key" = "value" |my key:my value |No
-|category:key = value |my-key:my-value |No
-|category:this.* = value |this.is.my.key:value |Yes
-|category:*my.key = value |this.is.my.key:value |Yes
-|category:this.* = value |this.is.my.key:some-value |No
-|category:this.* = some-* |this.is.my.key:some-value |No
-|===
+. From the Exchange landing page, select a filter option.
+. Click an option from the menu and click *Apply*.
+. To clear a filter option, click *Clear filter*.
+. To clear all filter options, click *Clear all filters*.
 
-You can search for a combination of different categories. You cannot search for multiple values of the same category.
+== Sort
 
-Multiple search terms use an implicit `AND`. `OR` is not supported.
+Assets can be sorted by the following options:
 
-These search terms return all assets marked as both system APIs and Salesforce products:
+* Relevance
++
+Sorts the matching results by where a match is found in the asset metadata, such as name, title, pages, and API specification metadata. API specification metadata includes endpoints and types.
```

Comment:
maybe add that this is only available when a search term is entered in the search box, and is the default when available

Reply:
fixed